services/sleeper/src/sleeper/main.py

```diff
@@ -0,0 +1,110 @@
+import os
+import json
+import random
+import time
+import subprocess
+
+
+def get_from_environ(key, default=None):
+    """Returns a value from the environ if not found None"""
+    return os.environ.get(key, default)
+
+
+def get_random_sleep():
+    """Returns a random amount of sleep between 1 and 9"""
+    return random.randint(1, 9)
+
+
+def cast_bool(value):
+    """Transformas a string into a boolean"""
+    return value.lower() in {"true", "yes", "1"}
+
+
+def validate_not_negative_int(number):
+    """Returns the number if >= 0 or a random int in range [1:9]"""
+    return number if number >= 0 else get_random_sleep()
+
+
+def test_gpu_cuda_code():
+    """Dose some computation on the GPU with CUDA"""
+    if get_from_environ("DISABLE_GPU_FOR_TESTING") is not None:
+        print("GPU payload disabled for testing")
```

Comment:
should we use log instead of print. If I recall well, the external logger parses stdout and looks for specific prefix. That is easier to control with loggers than with print

Reply:
I assumed that stdout behaviour would remain the same, as the previous C++ implementation used out, but I might be wrong.
This was migrated to proper logging. Dropped default log level to INFO.

Reply:
the sidecar does not parse the stdout in any way at the moment. it just takes everything created by the service.